snapcraft.yaml

```diff
@@ -941,6 +941,11 @@ parts:
       - --prefix=/usr
       - --with-builtin=pulse
     build-environment: *buildenv
+    override-build: |
+      set -eux
```

Comment:
override does this already before running this, or should at least

Reply:
`set -eux` is in effect without this line. I will drop this line. For consistency the same should be done for the other occurrences in a separate PR.